src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -67,6 +101,9 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
     ? getRangeOptions()
     : getPercentageOptions()
 
+  // TODO: add call bellow when the feature is implemented
+  // trackClick("We recommend changing your offer", priceOptions[0]?.value!)
```

Comment:
This should trigger when the "We recommend changing your offer to..." cta is implemented on [NX-3013].